tests/ludwig/modules/test_embedding_modules.py

```diff
@@ -19,6 +19,24 @@ def test_embed(
         embedding_size=embedding_size,
         representation=representation,
     )
+    inputs = torch.randint(0, 2, size=(2, 1)).bool()
+    outputs = embed(inputs)
+    assert outputs.shape[1:] == embed.output_shape
+
+
+@pytest.mark.parametrize('vocab', [['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h']])
+@pytest.mark.parametrize('embedding_size', [5, 10])
+@pytest.mark.parametrize('representation', ['dense', 'sparse'])
```

Comment:
maybe reduce these ranges like for the set ones?

Reply:
Which range? The embedding size?

Reply:
vocab too